config/schema_descriptions.yml

```diff
@@ -0,0 +1,72 @@
+    hypertension: "Yes if patient is hypertensive, no if not, and nil if the data is unavailable"
+    prior_heart_attack: "Yes if the patient has had a heart attack, no if not, nil if unknown"
+    prior_stroke: "Yes if the patient has had a stroke, no if not, nil if unknown"
+    chronic_kidney_disease: "Yes if the patient has had a chronic kidney disease, no if not, nil if unknown"
+    receiving_treatment_for_hypertension: "Yes if the patient was already receiving treatment for hypertension during registration"
+    diabetes: "Yes if patient is diabetic, no if not, and nil if the data is unavailable"
+    assigned_facility_id: "Facility ID to which the patient is currently assigned"
+    assigned_facility_size: "Size of the assigned facility"
+    assigned_facility_type: "Type of the assigned facility"
+    assigned_facility_slug: "Human readable ID of the assigned facility"
+    assigned_facility_region_id: "Region ID of the assigned facility"
+    assigned_block_slug: "Human readable ID of the assigned facility's block"
+    assigned_block_region_id: "ID of the assigned facility's block"
+    assigned_district_slug: "Human readable ID of the assigned facility's district"
+    assigned_district_region_id: "ID of the assigned facility's district"
+    assigned_state_slug: "Human readable ID of the assigned facility's state"
+    assigned_state_region_id: "ID of the assigned facility's state"
+    assigned_organization_slug: "Human readable ID of the assigned facility's organization"
+    assigned_organization_region_id: "ID of the assigned facility's organization"
+    registration_facility_id: "Facility ID to which the patient is currently registration"
+    registration_facility_size: "Size of the registration facility"
+    registration_facility_type: "Type of the registration facility"
+    registration_facility_slug: "Human readable ID of the registration facility"
+    registration_facility_region_id: "Region ID of the registration facility"
+    registration_block_slug: "Human readable ID of the registration facility's block"
+    registration_block_region_id: "ID of the registration facility's block"
+    registration_district_slug: "Human readable ID of the registration facility's district"
+    registration_district_region_id: "ID of the registration facility's district"
+    registration_state_slug: "Human readable ID of the registration facility's state"
+    registration_state_region_id: "ID of the registration facility's state"
+    registration_organization_slug: "Human readable ID of the registration facility's organization"
+    registration_organization_region_id: "ID of the registration facility's organization"
+    blood_pressure_id: "ID of the latest blood pressure as of this month. Use this to join with the blood pressures table."
+    bp_facility_id: "ID of the facility at which the latest BP was observed as of this month"
+    bp_recorded_at: "Time (in UTC) at which the latest BP as of this month was recorded"
```

Comment:
Some fragmentation of language across:
* BP vs blood pressure
* Observed vs recorded vs measured

Shall we consistent-ize this?

Reply:
Thanks! I've used "recorded" in all places, to also be consistent with the "recorded_at" field.